src/__experimental__/components/search/search.component.js

```diff
@@ -17,9 +17,25 @@ const Search = ({
   );
   let inputRef = useRef(null);
```

Comment:
Is seems that this ref is not used anywhere

Reply:
removed